src/SFA.DAS.Campaign.Web/Startup.cs

```diff
@@ -15,7 +15,7 @@
 using Sfa.Das.Sas.Core.Configuration;
```

Comment:
I'm getting exceptions thrown when navigating in this code:

https://github.com/SkillsFundingAgency/das-campaign/blob/43e18c941b1be66a66b505c532667f6ea222bbfe/src/SFA.DAS.Campaign.Web/Startup.cs#L181-L188

Reply:
i've uncommented the required if statement which was missed